flags.go

```diff
@@ -464,6 +464,9 @@ func populateFlags(c *cli.Context) (flags *flagStorage, err error) {
 
 	clientProtocolString := strings.ToLower(c.String("client-protocol"))
 	clientProtocol := mountpkg.ClientProtocol(clientProtocolString)
+	userStatCacheTTL := util.RoundDurationToNextMultiple(c.Duration("stat-cache-ttl"), time.Second)
```

Comment:
can we first take minimum and then call RoundDurationToNextMultiple? it will reduce one call to RoundDurationToNextMultiple.

Reply:
Good idea. Done.